### DISCUSSION TITLE
Windows 語音輸入功能 快捷鍵(win + H)

### DISCUSSION THREAD

**Discussion by JSHT** [1101-GS4719-C/Python-Community-GS4719-C#410] [General]
# win + H
## Intro
最近系統更新上 win 11 後，在尋找介紹 win 11 新功能的介紹時，找到了 [Windows 11 來了！今天就來把它的新功能榨得一滴不剩吧 ~](https://www.youtube.com/watch?v=Z2HYPD_XF40) 這部影片。裡面有提到一項內建的語音轉文字功能，實際使用下來準確度蠻高的，推薦大家可以玩玩看。

## How to use it on win 11
1. 設定 > 時間與語言 > 語言與地區 > 安裝語言 (有語音辨識符號的才可以使用，繁中預設就有)
![image](https://user-images.githubusercontent.com/48956859/156893968-5abb2d7a-4f23-4fbb-b37c-a8b37425e7c3.png)
2. 設定 > 系統 > 音效 > 設定音訊輸入麥克風
![image](https://user-images.githubusercontent.com/48956859/156894031-9fd8ca30-8263-4890-9084-7049a0bd6789.png)
3. 在任意可以輸入文字的地方按下 win + H，就可以開始使用了
![image](https://user-images.githubusercontent.com/48956859/156894379-05924e3e-df44-425a-a4bc-ad91ad6ffffc.png)

## YT影片中音檔轉文字
由於這項功能是需要使用麥克風收音後進行轉換，這方面可以透過使用一些虛擬音訊軟體來直接進行轉換。
這邊選用 [VBCable](https://vb-audio.com/Cable/) 進行
- 下載 VBCable
![image](https://user-images.githubusercontent.com/48956859/156894685-0b9d6072-a200-4e56-a1f9-96a5ecdd647b.png)
- 安裝 driver
![image](https://user-images.githubusercontent.com/48956859/156894747-86a57b69-d6ba-442e-a920-0a58a3dad3ee.png)
- 設定音訊輸入輸出
![image](https://user-images.githubusercontent.com/48956859/156894919-b87849ca-739a-4026-9b73-c688428a9408.png)
![image](https://user-images.githubusercontent.com/48956859/156894928-6d88ccaf-6101-4084-8723-79003c67dc5c.png)
- 撥放音檔後，在文字編輯器開啟語音輸入
以 [經典名場面](https://youtu.be/h8mqCy-KK0Y?t=22)、[繞口令](https://youtu.be/pOPCO2zqpLw?t=15) 來進行測試。
- output vs original text
經典名場面
`output: スターバースト。ストリーム。`
`original: starburst stream`
偷偷星爆
繞口令
`output:  斜め77°の並びで泣く泣く嘶くナナハン七台難なく並べてながながめ`
`original: 斜め七十七度の並びで泣く泣くいななく ナナハン七台難なく並べて長眺め`
(這邊日文發音兩者是相同的，差別在於有沒有轉換成漢字而已)

## 注意事項
1. 輸入的音檔是什麼語言，就需要調至對應的語言來使用這項功能。
![image](https://user-images.githubusercontent.com/48956859/156895144-0f5c74f0-c812-4153-b53d-97488a44dfc3.png)
2. 在轉換期間無法進行其他操作，否則會中斷執行。

最後如果哪裡有誤的話，麻煩在下方指正了~

**Comment by pekoranoinu** [comment]
我還沒升級到win11，因為我怕會有甚麼bug。

**Reply by JSHT** [reply]
原本沒要升，結果電腦出問題，想說乾脆直升。
